src/haz3lcore/statics/Info.re

```diff
@@ -318,9 +319,21 @@ let status_pat = (ctx: Ctx.t, mode: Mode.t, self: Self.pat): status_pat =>
    depending on the mode, which represents the expectations of the
    surrounding syntactic context, and the self which represents the
    makeup of the expression / pattern itself. */
-let status_exp = (ctx: Ctx.t, mode: Mode.t, self: Self.exp): status_exp =>
+let rec status_exp = (ctx: Ctx.t, mode: Mode.t, self: Self.exp): status_exp =>
   switch (self, mode) {
   | (Free(name), _) => InHole(FreeVariable(name))
+  | (InexhaustiveMatch(self), _) =>
```

Comment:
Translates `Self.exp.InexhaustiveMatch` to `Info.error_exp.InexhaustiveMatch`. Currently, `additional_err` can only be `Common(Inconsistent(Internal(_)))`.